runtime/dom-slot.js

```diff
@@ -73,13 +73,27 @@ class DomSlot extends Slot {
         });
       }
       if (content.model) {
+        if (arc) {
+          this._replaceViewDescriptions(content.model, arc);
+        }
+
         if (!this._liveDom) {
           this._stampTemplate(templates[templateName], eventHandler);
         }
         this._liveDom.set(content.model);
       }
     }
   }
+  _replaceViewDescriptions(model, arc) {
+    for (let key in model) {
+      let matchers = model[key].match && model[key].match(/^{{([a-zA-Z]*).description}}$/);
+      if (matchers && matchers.length > 1) {
+        let view = arc.findViewByName(this.particleSpec.particle.name, matchers[1]);
+        if (view && view.description)
+          model[key] = view.description;
```

Comment:
If the description is already attached to the view, couldn't the particle just do (simplified):
```js
_render(props, state) {
  return {
    choices: this.views.get('choices').description
  }
}
```  

?